src/CoreBundle/private/library/classes/dbobjects/TCMSImageEndpoint.class.php

```diff
@@ -733,11 +733,14 @@ public static function imagecopymerge_alpha($dst_im, $src_im, $dst_x, $dst_y, $s
      */
     public function GetRelativeURL()
     {
-        if (!$this->_isThumbnail) {
-            $sImageURL = $this->GetImageUrlPathPrefix().$this->aData['path'];
-        } else {
-            $sImageURL = URL_MEDIA_LIBRARY_THUMBS_PATH.$this->GetThumbPathExtension().'/'.$this->aData['path'];
+        /** @var string|null $dataPath */
+        $dataPath = $this->aData['path'] ?? '';
```

Comment:
In what case is aData a boolean?

I take it that is some earlier (access) error? It should probably be either catched already there or be explicitly checked here (as an early return?).

Reply:
aData is a empty array() in normally request. and empty array is false in PHP 7.4.
This case attached as image in the Issue.

Reply:
This changes have been reset, fix now directly in to the Load method.